semeio/jobs/rft/gendata_rft.py

```diff
@@ -158,17 +182,16 @@ def run(
                 well=well, time=time
             )
 
-            # Write trajectory and associated data to three  files,
+            # Write trajectory and associated data to ASCII files
             # individual pr. well and time, this is for GEN_DATA to pick up.
-            _write_gen_data_files(
-                trajectory_df,
-                os.path.join(outputdirectory, "RFT_{}_{}".format(well, report_step)),
-            )
+            _write_gen_data_files(trajectory_df, outputdirectory, well, report_step)
 
             # Aggregate dataframe for all wells and report steps.
             dframes.append(trajectory_df)
+        else:
+            logger.warning("No trajectory points loaded")
 
-    if csvfile is not None:
+    if csvfile is not None and dframes:
```

Comment:
Should we warn if the user has specified `csvfile` but one can not be provided? Or alternatively, move this below the check on line 174?

Reply:
Yes, it should be moved down.

Reply:
Or, maybe not. When the ValueError hits two lines down, it would be nice to have the CSV file for debugging.

Reply:
`and dframes` is now  reinstated, and a new test warrants its presence.